README.md

```diff
@@ -92,42 +86,105 @@ For testing, we have created a number of sample user credentials:
 
 ## Admin Access
 
-**TODO**: add superuser information for heroku. *not sure if django superuser is the same. Can we provide them with superuser login in that will work for them?
+**TODO**: add superuser information for heroku. \*not sure if django superuser is the same. Can we provide them with superuser login in that will work for them?
 
 When running locally, you can run this command to create a superuser and access the admin interface:
-	
-	python3 manage.py createsuperuser
-
+python3 manage.py createsuperuser
 
 ## API Call Formats:
 
-The format of requests are found in the example_requests folder (https://github.com/CMPUT404-W2020/CMPUT404-project-socialdistribution/tree/master/example_requests). 
+The format of requests are found in the example_requests folder (https://github.com/CMPUT404-W2020/CMPUT404-project-socialdistribution/tree/master/example_requests).
 
-**TODO**: update this 
+**TODO**: update this
 
 The system only allows GET and POST requests; all other requests will be responded to with an HTTP 405 response
 (will be updated to match social_distribution/sd/urls.py found on the api branch)
 
-    
-    auth/register               (POST)
-    auth/logout                 (GET)
-    auth/getuser                (GET)
-    auth/edituser/<uuid>        (POST)
-    auth/getpost                (GET)
-    auth/deletepost             (GET)
-    auth/getallpost             (GET)
-    posts/<uuid>                (GET)
-    posts/<uuid>/comments       (GET)
-    author/<uuid>/post          (POST)
-    author/<uuid>/posts         (GET)
-    author/posts                (GET)
-    posts/<uuid>/comment        (POST)
+    auth/register
+    - POST
+        - create user
+        {
+            first_name : fname,
+            last_name: lname,
+            username: uname,
+            password: password,
+            email : email
+        }
+
+    author/<uuid>
+    - GET
+        - returns author information
+        - uuid = author UUID
+
+    auth/edituser/<uuid>
+    - PUT
```

Comment:
line 100 says `The system only allows GET and POST requests`, there is also a DELETE request below